src/odemis/gui/cont/tabs.py

```diff
@@ -2919,65 +2926,18 @@ def __init__(self, name, button, panel, main_frame, main_data):
         panel.html_moi_doc.LoadPage(self.doc_path)
 
         if main_data.streak_cam:
+            self._settings_controller = settings.StreakCamAlignSettingsController(panel, tab_data)
```

Comment:
Let's call it `_streak_settings_cont(roller)`? This would make it clearer that it doesn't handle _all_ the settings.